src/fundus/parser/utility.py

```diff
@@ -567,7 +567,7 @@ def image_extraction(
         "(./ancestor::figure//*[(contains(@class, 'copyright') or contains(@class, 'credit')) and text()])[1]"
     ),
     author_filter: Optional[Pattern[str]] = None,
-    relative_urls: bool = False,
+    relative_urls: Union[bool, XPath] = False,
```

Comment:
I suggest changing this slightly, to cover unusual cases. Refer to `People` for an examplory use-case.